tests/selenium_test_websocket.py

```diff
@@ -0,0 +1,66 @@
+from selenium import webdriver
+
+def log_in(driver):
```

Comment:
You know, these functions are really common and can be used for testing other features in future? 
Can you please make class and put all the common functions and use from it? :smile: